src/core/a-animation.js

```diff
@@ -363,6 +364,8 @@ function getAnimationValues (el, attribute, dataFrom, dataTo, currentValue) {
     getForCoordinateComponent();
   } else if (['true', 'false'].indexOf(dataTo) !== -1) {
     getForBoolean();
+  } else if (dataTo && dataTo[0] === '#') {
```

Comment:
The THREE.Color class does it for you.
http://threejs.org/docs/#Reference/Math/Color
You can do ```new THREE.Color('red').getHexString()````